### PR TITLE
Fix tests for record file option parsing

### PR DESCRIPTION
**Pull request by hyades**

### CODE REVIEW

File: python-api/tests/integrationtests/test_controller.py [hunk 1]
```diff
@@ -376,10 +376,10 @@ def test_new_record(self):
 
                 curr_time = datetime.datetime.now()
                 alt_curr_time = curr_time + datetime.timedelta(0, 1)
-                time_str = curr_time.strftime('%Y-%m-%d %H%M%S')
```
Comment (mithro) [inline python-api/tests/integrationtests/test_controller.py]: Just use %Y.

Or you could use %q maybe instead?

File: python-api/gstswitch/server.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def __init__(
             video_port=3000,
             audio_port=4000,
             control_port=5000,
-            record_file='record.data'):
+            record_file='record-%Y-%m-%d_%H%M%S.data'):
```
Comment (mithro) [inline python-api/gstswitch/server.py]: This should be left empty by default I think.

`record_file=True` means `-r`, `record_file='record-%Y-%m-%d'` should be the same as `--record="record-%Y-%m-%d"`

**Comment by hyades** [comment]
@mithro updated commit.
Now, using a year format for integration tests. The default option in the Python API still remains to record in the year-month-date_time format.

**Comment by mithro** [comment]
I think https://github.com/timvideos/gst-switch/pull/117 conflicts with this pull request.

@hyades and @MaZderMind - can you guys figure out which one I should merge?

**Comment by hyades** [comment]
@mithro: https://github.com/timvideos/gst-switch/pull/117 is closed. You may merge this one.

**Comment by mithro** [comment]
LGTM - merging.

**Pull request merged**